src/justfile.rs

```diff
@@ -1,12 +1,14 @@
+use parallel::Ran;
+use std::sync::Arc;
```

Comment:
We can move this `use` into `src/lib.rs` so it doesn't need to be imported in multiple files.